odoo/addons/base/models/ir_rule.py

```diff
@@ -230,48 +203,40 @@ def _make_access_error(self, operation, records):
         resolution_info = _("Contact your administrator to request access if necessary.")
 
         if not self.env.user.has_group('base.group_no_one') or not self.env.user.has_group('base.group_user'):
-            msg = """{operation_error}
-
-{resolution_info}""".format(
-                operation_error=operation_error,
-                resolution_info=resolution_info)
-            return AccessError(msg)
+            return AccessError(f"{operation_error}\n\n{resolution_info}")
 
         # This extended AccessError is only displayed in debug mode.
         # Note that by default, public and portal users do not have
         # the group "base.group_no_one", even if debug mode is enabled,
         # so it is relatively safe here to include the list of rules and record names.
         rules = self._get_failing(records, mode=operation).sudo()
 
-        records_description = ', '.join(['%s (id=%s)' % (rec.display_name, rec.id) for rec in records[:6].sudo()])
+        records_sudo = records[:6].sudo()
+        company_issue = any('company_id' in (r.domain_force or '') for r in rules)
```

Comment:
That naming makes the information seem a lot more authoritative than it should be,

Reply:
What about `company_related` ?

Reply:
I will change it, thx.